tutoraspects/templates/aspects/jobs/init/dbt/init-dbt.sh

```diff
@@ -16,4 +16,4 @@ echo "Installing dbt dependencies"
 dbt deps --profiles-dir /app/aspects/scripts/aspects/
 
 echo "Running dbt"
-dbt run --profiles-dir /app/aspects/scripts/aspects/
+XAPI_SCHEMA={{ ASPECTS_XAPI_DATABASE }} ASPECTS_ENROLLMENT_EVENTS_TABLE={{ ASPECTS_ENROLLMENT_EVENTS_TABLE }} dbt run --profiles-dir /app/aspects/scripts/aspects/
```

Comment:
These changes aren't sustainable as sources grow, but work for now. We'll likely need a whole env vars file that we can source in or something.